docs/components/gltf-model.md

```diff
@@ -78,6 +78,16 @@ file.
 |----------|--------------------------------------|
 | selector | Selector to an `<a-asset-item>`      |
 | string   | `url()`-enclosed path to a glTF file |
+| Data URL | data:model/gltf-binary;base64,<data> |
```

Comment:
is `<data>` a typo?

Reply:
Not a typo, it's a placeholder, it means you replace it by the actually content of the base64 data of your model.

Reply:
If it's confusing to me it will be confusing to others too. above there is `<a-asset-item>`, yours doesn't refer to HTML. Can we put it in a wat it's not ambiguous?

Reply:
Okay, I replaced by "..."